packages/react-dom/src/events/SyntheticKeyboardEvent.js

```diff
@@ -54,6 +54,23 @@ var KeyboardEventInterface = {
     }
     return 0;
   },
+  code: function(event) {
+    // `code` represents a physical key on the keyboard, as opposed to the character
+    // generated by pressing the key.
+
+    // It returns a value not altered by keyboard layout
+    // or the state of the modifier keys.
+
+    // Assuming the input doesn't come from a physical keyboard, but instead from a virtual
+    // keyboard or accessibility device, it will be returned a value set by the browser to match
+    // as closely as possible to what would happen with a physical keyboard, to maximize
+    // compatibility between physical and virtual input devices.
+    if (event.type === 'keydown' || event.type === 'keyup') {
+      return event.code;
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code), `code` isn't widely supported. Is there a way to cheaply polyfill this for those browsers?